application/src/main/webapp/vue-app/components/ExoChatApp.vue

```diff
@@ -38,7 +38,7 @@
         @refresh-contacts="refreshContacts($event)" />
     </div>
     <div
-      v-show="(selectedContact && (selectedContact.room || selectedContact.user)) || mq === 'mobile'"
+      v-show="(isSelectedContact && selectedContact && (selectedContact.room || selectedContact.user)) || mq === 'mobile'"
```

Comment:
this is unneeded IMO ?
```suggestion
      v-show="(selectedContact && (selectedContact.room || selectedContact.user)) || mq === 'mobile'"
```